lib/faexport/scraper.rb

```diff
@@ -228,6 +228,65 @@ def journal_comments(id)
     comments("journal/#{id}/")
   end
 
+  def search_results(query, page)
+    # TODO: Make these part of the GET request, keep as defaults
+    perpage = 60              # 24, 36, 48, 60
+    order_by = 'date'         # relevancy, date, popularity
+    order_direction = 'desc'  # asc, desc
+    range = 'all'             # day, 3days, week, month, all
+    mode = 'extended'         # all, any, extended
+    rating_general = '&rating-general=on'
+    rating_mature = '&rating-mature=on'
+    rating_adult = '&rating-adult=on'
+    type_art = '&type-art=on'
+    type_flash = '&type-flash=on'
+    type_photo = '&type-photo=on'
+    type_music = '&type-music=on'
+    type_story = '&type-story=on'
+    type_poetry = '&type-poetry=on'
+
+    if page <= 1
+      page = 1
+      submit_mode = '&do_search=Search'
+    else
+      page = page - 1
+      submit_mode = '&next_page=>>> #{perpage} more >>>'
```

Comment:
Also, if you look at what's beeing sent to the server, it looks like you can just pass a parameter of `page=1` rather than messing with the next/prev buttons.
![2015-05-17-123246_398x205_scrot](https://cloud.githubusercontent.com/assets/11081061/7671772/dc1cab92-fc90-11e4-9a55-23744ef1d58a.png)